packages/drivers/odsp-driver/src/createFile.ts

```diff
@@ -38,7 +38,7 @@ const isInvalidFileName = (fileName: string): boolean => {
 };
 
 /**
- * Creates a new Fluid file. '.fluid' is appended to the filename
+ * Creates a new Fluid file. If no extension is supplied then use .fluid as extension.
```

Comment:
I think this comment is incorrect. If no extension is provided, then no extension

Reply:
Yes was in process of removing the old changes